expression/builtin_string.go

```diff
@@ -3073,7 +3073,10 @@ func (c *exportSetFunctionClass) getFunction(ctx sessionctx.Context, args []Expr
 	if err != nil {
 		return nil, err
 	}
-	bf.tp.Flen = mysql.MaxBlobWidth
+	// Max length is (64 * 2 -1) * 4
+	// For example:
+	// "Y,Y,Y,Y,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N,N"
+	bf.tp.Flen = 508
```

Comment:
What will happen if arg[1]/arg[2] is too long? 
I think using the formula in mysql is better.. https://github.com/mysql/mysql-server/blob/8.0/sql/item_strfunc.cc#L3646

Reply:
Any example？

Reply:
It seems that MySQL always return 508.
``` sql
mysql> SELECT EXPORT_SET(5,'Y','N',',',4);
Field   1:  `EXPORT_SET(5,'Y','N',',',4)`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       VAR_STRING
Collation:  utf8mb4_0900_ai_ci (255)
Length:     508
Max_length: 7
Decimals:   31
Flags:


+-----------------------------+
| EXPORT_SET(5,'Y','N',',',4) |
+-----------------------------+
| Y,N,Y,N                     |
+-----------------------------+
1 row in set (0.00 sec)
```

Reply:
OK, I will fix it.